src/components/AdminSettings/MatterbridgeIntegration.vue

```diff
@@ -138,12 +143,17 @@ export default {
 				this.matterbridgeVersion = response.data.ocs.data.version
 				this.matterbridgeEnabled = true
 				this.saveMatterbridgeEnabled()
-
-				this.isInstalling = false
+				this.errorText = ''
 			} catch (e) {
-				console.error(e)
 				showError(t('spreed', 'Failed to execute Matterbridge binary.'))
+				if (e.response.data.ocs.data.error === 'binary_permissions') {
```

Comment:
this should always show? Not only after clicking the enable/install button?

Reply:
Right, done.